features/step_definitions/project.rb

```diff
@@ -34,6 +34,47 @@
   end
 end
 
+Given /^I have a privileged project$/ do
+  # system projects should not be selected by default
+  sys_projects = BushSlicer::Project::SYSTEM_PROJECTS
+
+  project = @projects.reverse.find {|p|
+    !sys_projects.include?(p.name) &&
+      p.is_user_admin?(user: user, cached: true) &&
+      p.active?(user: user, cached: true)
+  }
+  if project
+    # project does exist as visible is doing an actual query
+    # also move project up the stack
+    @projects << @projects.delete(project)
+  else
+    projects = (user.projects - sys_projects).select {|p|
+      p.is_user_admin?(user: user, cached: true) &&
+      p.active?(user: user, cached: true)
+    }
+    if projects.empty?
+      step 'I create a new project'
+      unless @result[:success]
+        logger.error(@result[:response])
+        raise "unable to create project, see log"
+      end
+    else
+      cache_resources *projects
+    end
+  end
+  if env.version_ge("4.12", user: user)
```

Comment:
@Phaow  If we only add this for 4.12+, the step name makes confused for the previous release, right?

Reply:
@duanwei33 Yeah. TBH, I don't have a better step name for this. How about `Given /^I have a project using hostpath`?

Reply:
@Phaow  I agree it is not easy to backport.
From the code side, I prefer to apply them on all releases (if this cmd works), and let the user choose to use it or not in different releases. So the code and the step name are consistent.
Like in 4.11-, users could use "I have a project" if they don't need a privileged one.

Reply:
Adding @liangxia may give us some suggestions here.

Reply:
Sure. Thanks!

Reply:
@Phaow Another way(I think it is better one) is we could add an individual step like "labeling () project as pod-security privileged", then we could use this step after "I have a project".

Reply:
Good option! I'll have a try.

Reply:
How about `Given /^I have a project with proper privilege$/` ?
```
Given /^I have a project with proper privilege$/ do
  step %Q/I have a project/
  if env.version_ge("4.12", user: user)
    blabla
  end
end

Reply:
Perfect. Many thanks!